grails-app/views/finder/index.gsp

```diff
@@ -131,7 +131,9 @@
                 </a>
             </div>
 
-            <p>IMOS is a national collaborative research infrastructure, supported by Australian Government. It is operated by a consortium of institutions as an unincorporated joint venture, with the <a target="_blank" class="external" title="UTAS home page" href="http://www.utas.edu.au/">University of Tasmania</a> as Lead Agent.
+            <p>Australia’s Integrated Marine Observing System (IMOS) is enabled by the National Collaborative Research Infrastructure Strategy (NCRIS).
+                It is operated by a consortium of institutions as an unincorporated joint venture,
```

Comment:
Missing "with the". Should read:
It is operated by a consortium of institutions as an unincorporated joint venture, with the University of Tasmania as Lead Agent.